src/Time/TimeSteppers/AdamsBashforthN.hpp

```diff
@@ -369,7 +375,33 @@ void AdamsBashforthN::update_u(
     const gsl::not_null<Vars*> u,
     const gsl::not_null<History<Vars, DerivVars>*> history,
     const TimeDelta& time_step) const noexcept {
-  update_u_impl(u, *history, time_step);
+  update_u_impl(u, *history, time_step, order_);
+  history->mark_unneeded(history->begin() + 1);
+}
+
+template <typename Vars, typename DerivVars>
+void AdamsBashforthN::update_u(
+    const gsl::not_null<Vars*> u,
+    const gsl::not_null<std::optional<Vars>*> u_error,
+    const gsl::not_null<History<Vars, DerivVars>*> history,
+    const TimeDelta& time_step) const noexcept {
+  if(order_ == 1) {
+    ERROR(
+        "Cannot reliably estimate the error for an order = 1 Adams-Bashforth "
+        "scheme. Consider using a higher order for the time-stepper.");
```

Comment:
AB0 (i.e., dy=0) is useless for an actual integration, but might be good enough for the error estimate.  I don't know.